src/alchemtest/amber/access.py

```diff
@@ -96,12 +97,57 @@ def load_invalidfiles():
         - 'data' : the example of invalid data files
         - 'DESCR': the full description of the dataset
 
+
+    .. deprecated:: 0.7
+       use :func:`load_testfiles` instead
+
+    """
+
+    import warnings  # NOTE: this import is here and not where it should be just because we'll remove this function in next version
+    warnings.warn(
+        "load_invalidfiles() was deprecated in 0.7.0 and will be removed in the following release."
+        " Use load_testfiles() instead",
+        DeprecationWarning)
+    module_path = dirname(__file__)
+    data = [[
+        glob(join(module_path, 'testfiles', 'no_useful_data.out.tar.bz2'))[0],
+        glob(join(module_path, 'testfiles', 'no_control_data.out.tar.bz2'))[0],
+        glob(join(module_path, 'testfiles', 'no_temp0_setted.out.tar.bz2'))[0],
+        glob(join(module_path, 'testfiles', 'no_free_energy_info.out.tar.bz2'))[0],
+        glob(join(module_path, 'testfiles', 'no_atomic_section.out.tar.bz2'))[0],
+        glob(join(module_path, 'testfiles', 'no_results_section.out.tar.bz2'))[0]]]
+
+    with open(join(module_path, 'testfiles', 'descr_invalid.rst')) as rst_file:
+        fdescr = rst_file.read()
+
+    return Bunch(data=data,
+                 DESCR=fdescr)
+
+
+def load_testfiles():
+    """Load incomplete or wrongly formatted files to be used to test the AMBER parsers.
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files
+        - 'DESCR': the full description of all the files
+
     """
 
     module_path = dirname(__file__)
-    data = [glob(join(module_path, 'invalidfiles/*.out.bz2'))]
 
-    with open(join(module_path, 'invalidfiles', 'descr.rst')) as rst_file:
+    data = {}
+    for f in listdir(join(module_path, 'testfiles')):
+        f_path = Path(f)
+        if f_path.suffix==".bz2":
+            while f_path.suffix in ('.tar', '.bz2', '.out'):
+                f_path = f_path.with_suffix('')
```

Comment:
Why do you do
```
        if f_path.suffix==".bz2":
            while f_path.suffix in ('.tar', '.bz2', '.out'):
                f_path = f_path.with_suffix('')
```
?

Reply:
Because I like the `key` of the dictionary to be just the base name of the file, without its extension, and in this way the extension is safely removed, being it `.out`, `.tar.bz2`, `.bz2`, `out.bz2`.